charts/sam/values.yaml

```diff
@@ -53,7 +53,17 @@ probes:
       periodSeconds: 10
       failureThreshold: 30 # 5 minutes before restarted
       successThreshold: 1
-
+  startup:
+    # probes.startup.enable -- Whether to configure a startup probe
+    enabled: false
```

Comment:
@choover-broad I noticed that the liveness probe wasn't enabled here. What are your thoughts?

Reply:
I believe it was disabled intentionally in order to prevent interruptions to liquibase migrations! So, I think we should enable it in this PR, too.

Reply:
argh, I think we'll need to revert the changes for ontology and firecloud orchestration as well, as they don't actually use liquibase. Apologies!

Reply:
e732039

Reply:
dc66690